### PR TITLE
Release/1.4.1

### PR DESCRIPTION
**Pull request by dariuszbolkowski**
Creating new release.
Adding support for the new version of NASK EZDRP application 19.7.15
Adding new version of images for 
- ezd_crd: rabbitmq, redis, postgresql
- ezd_backend:  rabbitmq, redis

### CODE REVIEW

**Review by qwiatu-lp — COMMENTED** [review]
@dariuszbolkowski I have added some comments

File: charts/ezd-backend/1.4.1/questions.yaml [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+    label: Redis Image Name
+    description: "Redis Image Name."
+    type: string
+    default: "quay.io/linuxpolska/ezd-backend_redis:7.0.13-alpine-3.15-r1"
+    show_if: "global.redis.deploy=true"
+  - variable: redisConfig.customConfig.kubernetesConfig.redisExporter.image
+    label: Redis Exporter Image Name
+    description: "Redis Exporter Image Name."
+    type: string
+    default: "quay.io/linuxpolska/ezd-backend_redis-exporter:v1.44.0"
```
[the remark this answers is not in the document]
Comment (dariuszbolkowski) [reply charts/ezd-backend/1.4.1/questions.yaml]: fixed

File: charts/ezd-backend/1.4.1/values.yaml [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+    storage:
+      volumeClaimTemplate:
+        spec:
+          accessModes: ["ReadWriteOnce"]
+          resources:
+            requests:
+              storage: 1Gi
+    redisExporter:
+      enabled: false
+      image: quay.io/linuxpolska/ezd-backend_redis-exporter:v1.44.0
```
[the remark this answers is not in the document]
Comment (dariuszbolkowski) [reply charts/ezd-backend/1.4.1/values.yaml]: fixed

File: charts/ezd-crd/1.4.1/values.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      cpu: 200m
+      memory: 500Mi
+  tolerations: []
+  affinity: {}
+
+
+cloudnative-pg:
+  image:
+    repository: quay.io/linuxpolska/ezd-crd_cloudnative-pg
+    tag: v1.22.2-debian-11-r1
```
[the remark this answers is not in the document]
Comment (dariuszbolkowski) [reply charts/ezd-crd/1.4.1/values.yaml]: fixed

**Pull request merged**